src/libstd/collections/mod.rs

```diff
@@ -85,6 +85,56 @@
 //! or "most important" one at any given time.
 //! * You want a priority queue.
 //!
+//! # Performance
+//!
+//! Choosing the right collection for the job requires an understanding of what each collection
+//! is good at. Here we briefly summarize the performance of different collections for certain
+//! important operations. For further details, see each type's documentation.
+//!
+//! Throughout the documentation, we will follow a few conventions. For all operations,
+//! the collection's size is denoted by n. If another collection is involved in the operation, it
```

Comment:
Yeah I wasn't sure about this. *Ideally* I would use mathmode, but we don't support it. Making all the math codemode is a bit weird looking.

* O(min(i, n-i))
* `O(min(i, n-i))`
* *O(min(i, n-i))*
* O(min(*i*, *n-i*))